tests/unit/query_test.py

```
@@ -245,4 +248,127 @@ def test_query_payment_requirement_defaults_to_true(query_requires_payment):
     query = Query()
     assert query._is_payment_required() == True
     # Verify that payment-requiring query also defaults to requiring payment
-    assert query_requires_payment._is_payment_required() == True
\ No newline at end of file
+    assert query_requires_payment._is_payment_required() == True
+
+@pytest.mark.parametrize(
+    'valid_amount,expected',
```

Ruff (0.14.10)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)